.github/workflows/push-to-iis.yaml

```
@@ -0,0 +1,38 @@
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v5
+        with:
+          name: 'iis-site'
+
+      - name: Stop IIS
+        run: Stop-IISSite -Name "terra-stack-docs"
+        shell: powershell
```

actionlint (1.7.7)

shell name "powershell" is invalid on macOS or Linux. available names are "bash", "pwsh", "python", "sh"

(shell-name)